lib/parity/environment.rb

```diff
@@ -132,7 +132,7 @@ def run_migration?
     end
 
     def rails_app?
-      Kernel.system("command -v rake && rake -n db:migrate")
+      Kernel.system('[ -d "db/migrate" ] && [ -f "Rakefile" ]')
```

Comment:
I think a `Rakefile` and a `db/migrate` directory is enough evidence to justify running `rake db:migrate`.